### ISSUE TITLE
Aria-live "polite" not reading at all

### ISSUE THREAD

**Issue by nattri** [nvaccess/nvda#12464]
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)

### Steps to reproduce:
1. Insert new html element to the DOM having aria-live="polite" property
2. Screen reader should announce newly added element as soon as user is idle.

### Actual behavior:
Newly added element is never announced

### Expected behavior:
Should announce newly element with as soon as user is idle (expected behaviour of arial-live="polite")

#### NVDA installed/portable/running from source:
Installed on windows 10 machine

#### Name and version of other software in use when reproducing the issue:
Google chrome - Version 90.0.4430.212

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by cary-rowen** [comment]
Hi,

Have you checked the status of NVDA Plus 5?

Thanks!

**Comment by nattri** [comment]
> NVDA Plus 5

Hi,

I don't know what that is but, I have download latest NVDA from nvaccess.org.

**Comment by seanbudd** [comment]
Hi @nattri. "NVDA+5" refers to pressing your NVDA key and 5. Usually the NVDA key is mapped to capslock and/or insert.

**Comment by ChilupuriAnilReddy** [comment]
Do we have solution for this. Facing this issue now.

**Comment by owenniblock** [comment]
I've also seen this issue, please let me know if additional reproduction steps would be helpful.

**Comment by Coldknife2** [comment]
Same, having the issue also

**Comment by owenniblock** [comment]
I do wonder if this is a bug or intended behaviour. Since the content of the element isn't changing (the whole element changes) perhaps not announcing the content is intentional (the same as when a page loads with existing content inside aria-live containers 🤔 

To work around this issue, we've created an aria-live element that exists on every page and we update that with our updates. It's not perfect but it gets around this issue (which is especially common when we're using React and a whole section of the DOM gets swapped out.

**Comment by Adriani90** [comment]
Closing as invalid, per last comment by @ramoncorominas. If you are having still any other arguments why this should not be the expected behavior, please comment and we can reopen.

**Comment by Adriani90** [comment]
See also related issues #15857 and #13774.
